tailwind.config.ts

```diff
@@ -4,6 +4,8 @@ export default {
   theme: {
     extend: {
       colors: {
+        pending: '#ffeb3b',
```

Comment:
I would put this under bcGovColor if they are going to be specific colors.
Otherwise it looks like its special entire class when used in tailwind e.g.
`bg-pending`
vs 
`bg-bcGovColor-pending`

we also did similar in the [BTR project common tailwind config](https://github.com/bcgov/business-transparency-registry/blob/main/btr-web/btr-common-components/tailwind.config.ts):

but what would probably be the best is, if it could be shade of e.g. yellow or smth similar.
It might fit under the yellow-200 if we are not using that one already